src/models/base.ts

```diff
@@ -4,7 +4,7 @@ import { guard } from '../utils';
 import { JSONObject, JSONValue } from '../utils/types';
 import { Signature } from './signature';
 
-const SPECIFICATION_VERSION = ['1', '20', '30'];
+const SPECIFICATION_VERSION = ['1', '0', '30'];
```

Comment:
Nit: the current version of the [specification is 1.0.31](https://theupdateframework.github.io/specification/v1.0.31/index.html), the only notable change from 1.0.30 is that we simplified how we describe the root rollback check in 5.3.5 of the detailed client workflow (see https://github.com/theupdateframework/specification/commit/7b565c71c4b6b95a303dc75f7efbfab7489e53c4).